packages/js/src/components/modals/editorModals/EditorModal.js

```diff
@@ -10,9 +11,31 @@ import Modal from "../Modal";
  *
  * Warning: contains styling that is specific for the Sidebar.
  *
- * @returns {*} A button wrapped in a div.
+ * @param {string} id The unique identifier for the modal.
+ * @param {string} postTypeName The post type label in singular form.
+ * @param {React.ReactNode} children The modal content.
```

Comment:
Does this also cover the array case?

Reply:
Yes, it is a recursive via `Iterable<ReactNode>`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L427